src/main/java/org/hawkular/client/android/backend/BackendClient.java

```diff
@@ -136,14 +136,15 @@ public void configureCommunication(@NonNull String host,
 
     private void configurePipes(URL backendUrl, Persona persona) {
         URL pipeUrl = Urls.getUrl(backendUrl, BackendPipes.Paths.ROOT);
-
+        URL alertReslveUrl = Urls.getUrl(pipeUrl, BackendPipes.Paths.ALERT_RESOLVE);
```

Comment:
Name have been changed. Should be changed any minute.
And was needed to changed as Here we are defining property of Pipe(Aerogear) And for that we need to set to which URL this PUT will work upon.